docs/getting-started/release-notes/v2_11_0.md

```diff
@@ -47,17 +48,16 @@ Zowe Version 2.11.0 contains the bug fixes that are described in the following t
 
 #### Zowe CLI (Core)
 
-#### Zowe CLI Imperative Framework
-
-#### CICS Plug-in for Zowe CLI
-
-#### DB2 Plug-in for Zowe CLI
+- Fixed an issue in the Daemon server which prevents users on Windows with uppercase letters in their username from using the Daemon.
+- Add check for invalid block size when creating a sequential data set. ([#1439](https://github.com/zowe/zowe-cli/issues/1439))
+- Allowed `logout` operations with invalid and/or expired tokens. ([#1734](https://github.com/zowe/zowe-cli/pull/1734))
+- Prevented misleading `basePath error` when credentials are invalid. ([#1734](https://github.com/zowe/zowe-cli/pull/1734))
```

Comment:
@adam-wolfe what do we mean by "prevented" here? Does the message not display any more? Or was the message updated? Or something else?

Reply:
I am not sure. @zFernand0 should know more.

Reply:
The message was technically incorrect. 
In this instance, it should have not needed a `basePath` since the user just wanted to logout, which goes to a separate endpoint that does not require `basePath` to be set.